tests/layer_tests/ovc_python_api_tests/test_ovc_cli_tool.py

```diff
@@ -74,6 +74,31 @@ def create_tf_model(self, tmp_dir):
         # save model to .pb and return path to the model
         return save_to_pb(tf_net, tmp_dir)
 
+    def create_tf_saved_model_dir(self, temp_dir):
+        import tensorflow as tf
+
+        input_names = ["Input1", "Input2"]
+        input_shape = [1, 2, 3]
+
+        x1 = tf.keras.Input(shape=input_shape, name=input_names[0])
+        x2 = tf.keras.Input(shape=input_shape, name=input_names[1])
+        y = tf.nn.sigmoid(tf.nn.relu(x1 + x2))
+        keras_net = tf.keras.Model(inputs=[x1, x2], outputs=[y])
+
+        tf.saved_model.save(keras_net, temp_dir + "/model")
```

Comment:
nit: better to use non-default name, for example, "/test_model"

Reply:
Done.